src/Employer/Employer.Web/Controllers/Part2/TrainingProviderController.cs

```diff
@@ -15,22 +16,28 @@ namespace Esfa.Recruit.Employer.Web.Controllers.Part2
     public class TrainingProviderController : Controller
     {
         private readonly TrainingProviderOrchestrator _orchestrator;
-        private readonly IEmployerVacancyClient _client;
         private readonly IRecruitVacancyClient _vacancyClient;
         private const string TrainingProviderJourneyTempDataKey = "FromSelectTrainingProvider";
         private const string InvalidUkprnMessageFormat = "The UKPRN {0} is not valid or the associated provider is not active.";
+        private const string InvalidSearchTerm = "Please enter a training provider name or UKPRN";
 
-        public TrainingProviderController(TrainingProviderOrchestrator orchestrator, IEmployerVacancyClient client, IRecruitVacancyClient vacancyClient)
+        public TrainingProviderController(TrainingProviderOrchestrator orchestrator, IRecruitVacancyClient vacancyClient)
```

Comment:
IRecruitVacancyClient is not used anywhere

Reply:
👍